docs/src/pages/components/image/examples/ImageObjectFitandPositionExample.tsx

```diff
@@ -0,0 +1,15 @@
+import { Image } from '@aws-amplify/ui-react';
```

Comment:
It seems we have duplicate files with different casing: 

```
$ git clone git@github.com:aws-amplify/amplify-ui.git

warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'docs/src/pages/components/image/examples/ImageObjectFitAndPositionExample.tsx'
  'docs/src/pages/components/image/examples/ImageObjectFitandPositionExample.tsx'
```

Can you look to remove one of the duplicates?